README.md

```diff
@@ -36,7 +36,6 @@ const fadeIn = [{ opacity: 0 }, { opacity: 1 }] // WAAPI keyframes
 const fadeOut = [{ opacity: 1 }, { opacity: 0 }] // WAAPI keyframes
 const animatedText = $$(".animated-text") // $$ ≈ querySelectorAll, use $ for querySelector
 
-// <span hidden class="animated-text"></span>
```

Comment:
Lol, there are supposed to be two spans to show how the $$() interface allows you to manipulate multiple elements at once.

But, I'm glad you flagged this because it's obvious how confusing it is in retrospect. Any idea how I can make it more clear that this is operating on several elements at once? I've been thinking about changing this into a full HTML file with a couple other examples because I think this is maybe a bit too basic and doesn't highlight some of the cooler asynchronous features that I talk about further down in the README.

Reply:
aaah got it. i think the simplest and safest way would be adding an unique attr to them. maybe data-item=1 and 2  but i would prefer adding more generic attributes like classes: eg: "item-1" and "item-2" classes or ids.

a website would be cool but it will be 100% needed if this gets traction. so it would be best to wait a bit and then find the optimal solution.(i would love to get involved btw)

Reply:
Sounds good! I have some plans for reactivity based off things like data attributes, so I think I lean in that direction so I can try to show that off when I'm finished. If you want, go ahead and update your commit and I'll merge your pull request. I'm leaning towards something like this:

```html
<span hidden class="animated-text" data-index="1"></span>
<span hidden class="animated-text" data-index="2"></span>
```

or 

```html
<span hidden class="animated-text" data-depth="1"></span>
<span hidden class="animated-text" data-depth="2"></span>
```